lib/src/presentation/view/card_list.dart

```diff
@@ -0,0 +1,351 @@
+                  color: Colors.black,
+                  child: const Text(
+                    kCardListMurlocs,
+                    textAlign: TextAlign.center,
+                    style: TextStyle(
+                      fontSize: cardTextInfoFontSize,
+                      fontWeight: FontWeight.bold,
+                    ),
+                  ),
+                ),
+                onTap: () {
+                  changeUrl(kRacesMurlocEndPointApi);
+                },
+              ),
+              ListTile(
+                title: Title(
+                  color: Colors.black,
+                  child: const Text(
+                    kCardListDemons,
+                    textAlign: TextAlign.center,
+                    style: TextStyle(
+                      fontSize: cardTextInfoFontSize,
+                      fontWeight: FontWeight.bold,
+                    ),
+                  ),
+                ),
+                onTap: () {
+                  changeUrl(kRacesDemonEndPointApi);
+                },
+              ),
+            ],
+          ),
+        ),
+        body: SingleChildScrollView(
+          child: Center(
+            child: DecoratedBox(
+              decoration: const BoxDecoration(
+                image: DecorationImage(
+                  image: AssetImage(
+                    assetPathBackgroundImage,
+                  ),
+                  fit: BoxFit.cover,
+                ),
+              ),
+              child: Column(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.only(
+                      left: cardTextPaddingLeftTopRight,
+                      top: cardTextPaddingLeftTopRight,
+                      right: cardTextPaddingLeftTopRight,
+                      bottom: cardTextPaddingBottom,
+                    ),
+                    child: Container(
+                      constraints: const BoxConstraints(
+                        minHeight: kCardListBoxMinHeight,
+                      ),
+                      child: StreamBuilder<CardEvent>(
+                        stream: widget.cardBloc.cardListStream,
+                        initialData: CardEvent(cardState: CardState.loading),
+                        builder: (
+                          BuildContext context,
+                          AsyncSnapshot<CardEvent> snapshot,
+                        ) {
+                          switch (snapshot.data!.cardState) {
+                            case CardState.loading:
+                              return const Center(
+                                child: CircularProgressIndicator(
+                                  strokeWidth: kCardListProgressIndStrokeWidth,
+                                  backgroundColor: Colors.black,
+                                  color: Colors.red,
+                                ),
+                              );
+                            case CardState.empty:
```

Comment:
Could you create a simple UI to show a message with this empty state?

Reply:
Done.